src/main/java/CreateDeadlineException.java

```diff
@@ -0,0 +1,5 @@
+public class CreateDeadlineException extends CreateTaskException {
```

Comment:
Should the class names be a noun? I noticed the same issue for your 4 Exception classes.

Reply:
Thank you for your insightful comments. I will review your feedback and implement the changes in a future commit !
:smile: